website/docs/tutorials/07-testing/README.md

```diff
@@ -99,7 +111,25 @@ func test_something():
 end
 ```
 
+```cairo title="Using setup_* case hook"
+%lang starknet
+
+@external
+func setup_test_something():
+    %{ max_examples(500) %}
```

Comment:
Maybe `deploy_contract` instead of `max_examples` would make this example simpler?
```suggestion
    %{ max_examples(500) %}
```

Reply:
`deploy_contract` does make a lot of sense within setup case, you can put it inside test case directly and this will work.

`max_examples` instead is `setup` specific and would often be used, hence this example.

Reply:
Yea, deploy contract is a bit weird. I just have a feeling that `max_examples` is not a simple enough cheatcode for an example, as you have to know what fuzzing is beforehand which makes learning curve steeper. 

Alternative idea, which _maybe_ tells a better story
```
    func __setup__():
        deploy_contract()
    end

     func setup_test_a():
        store(5)
    end

     func test_a():
        assert load() = 5
    end

     func test_b():
        assert load() = 0
    end
```

leaving the decision to you

Reply:
#726 I very very like your idea for an example. Let's do this later in separate PR.